Failed tests (12121): TestSplitSnapshotRace_SnapshotWins

The following test appears to have failed:

[#12121](https://circleci.com/gh/cockroachdb/cockroach/12121):

```
W0202 01:37:55.224217 640 storage/replica.go:1211  group 2 on store 4 failed to send message to 3: storage/raft_transport.go:163: unknown peer 3
W0202 01:37:55.232986 640 storage/replica.go:1211  group 11 on store 6 failed to send message to 2: storage/raft_transport.go:163: unknown peer 2
W0202 01:37:55.332923 640 storage/replica.go:1211  group 11 on store 6 failed to send message to 2: storage/raft_transport.go:163: unknown peer 2
W0202 01:37:55.432906 640 storage/replica.go:1211  group 11 on store 6 failed to send message to 2: storage/raft_transport.go:163: unknown peer 2
W0202 01:37:55.532796 640 storage/replica.go:1211  group 11 on store 6 failed to send message to 2: storage/raft_transport.go:163: unknown peer 2
panic: test timed out after 5m0s

goroutine 49411 [running]:
testing.startAlarm.func1()
	/tmp/workdir/go/src/testing/testing.go:703 +0x16b
created by time.goFunc
	/tmp/workdir/go/src/time/sleep.go:129 +0x6e

goroutine 1 [chan receive, 4 minutes]:
testing.RunTests(0x1d13c08, 0x24523a0, 0xe1, 0xe1, 0x1)
	/tmp/workdir/go/src/testing/testing.go:562 +0xafa
testing.(*M).Run(0xc8202eaf10, 0x1)
	/tmp/workdir/go/src/testing/testing.go:494 +0xe5
github.com/cockroachdb/cockroach/util/leaktest.TestMainWithLeakCheck(0xc8202eaf10)
	/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:37 +0x2f
github.com/cockroachdb/cockroach/storage_test.TestMain(0xc8202eaf10)
	/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:36 +0x34
main.main()
	github.com/cockroachdb/cockroach/storage/_test/_testmain.go:510 +0x20a

goroutine 17 [syscall, 4 minutes, locked to thread]:
runtime.goexit()
	/tmp/workdir/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 5 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x279afa0)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x76
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0x117

goroutine 13926 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820269820)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13914 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8208ab980)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13909 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc8208ab7a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12891 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:645 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc820386ae0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13896 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820268f20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12904 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc8205133c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12929 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbb908, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8209084c0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8209084c0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820908460, 0x0, 0x7f05b9fbbff0, 0xc820268c40)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc8200264e8, 0xc82046bd20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8208b41b0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13900 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:645 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc820268fa0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12884 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8204e6910)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 13923 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:645 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8202693c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13895 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbb488, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820139d40, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820139d40, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820139ce0, 0x0, 0x7f05b9fbbff0, 0xc820540b40)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026b18, 0xc82051ad20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820872ff0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13898 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:566 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc820268f60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13899 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:626 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc820268f80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12906 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc820513400)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12942 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8204e6280)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 12905 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc8205133e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12908 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc820513460)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12909 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc82090cc60, 0xece41fe69, 0x346d6a88, 0x279aa20, 0xc820510510, 0xc820526a10, 0xc8208c5e10, 0x62773c)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc8208c5e10, 0xc820510f90)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:214 +0xf6
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc8205110e0, 0xc820266a80)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:221 +0x46b
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:216 +0x2fe
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc820513480)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12883 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc8200d6780)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:214 +0xf1a
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:201 +0x2e
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820233970)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13907 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc8208ab760)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12935 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbbcc8, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820908140, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820908140, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc8209080e0, 0x0, 0x7f05b9fbbff0, 0xc82031c580)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026070, 0xc82046fd20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8208b4720)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12888 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1591 +0xb48
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc82086b620)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13019 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbb788, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820526ca0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820526ca0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820526c40, 0xc8208c3000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200ea8a0, 0xc8208c3000, 0x1000, 0x1000, 0x7f05b9fc77d0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc820901020)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 13951 [IO wait]:
net.runtime_pollWait(0x7f05b9fbb248, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8203c0610, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8203c0610, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8203c05b0, 0xc8204f1000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200ea4c8, 0xc8204f1000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc820835080, 0x7f05b9fc72b8, 0xc8200ea4c8, 0x7f05b9fc7520, 0xc820286a80, 0xc8209532b0, 0xc8202ce7e0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:251 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:254 +0xadc

goroutine 13938 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc8208353e0, 0xece41fe6a, 0x3198a43f, 0x279aa20, 0xc820510510, 0xc8200522a0, 0xc82048bad0, 0x62773c)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc82048bad0, 0xc820873ef0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:214 +0xf6
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc8205840c0, 0xc8205ace00)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:221 +0x46b
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:216 +0x2fe
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269c60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12887 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820386a40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13904 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1591 +0xb48
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc820135af0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12933 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8204eca00)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 13908 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc8208ab780)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13947 [IO wait]:
net.runtime_pollWait(0x7f05b9fbb6c8, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820578ae0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820578ae0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820578a80, 0xc8205c5000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820026598, 0xc8205c5000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc820835740, 0x7f05b9fc72b8, 0xc820026598, 0x7f05b9fc7520, 0xc820286780, 0xc820953040, 0xc8202ce600)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:251 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:254 +0xadc

goroutine 13017 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc820556120, 0x7f05b9fc7520, 0xc82087b800, 0xc820900f60)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:341 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc820556120, 0x7f05b9fc7520, 0xc82087b800, 0xc820900f60)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:248 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0xa6c

goroutine 13936 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269c20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12946 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820387f60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13905 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:566 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc820269380)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12965 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8204e7c70)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 12918 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8204e74f0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 13903 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820269300)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13937 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269c40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13928 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:566 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269860)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13917 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8208aba00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12903 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc8205133a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13018 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbb9c8, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820526d10, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820526d10, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820526cb0, 0xc820441000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200ea8f8, 0xc820441000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc820556120, 0x7f05b9fc72b8, 0xc8200ea8f8, 0x7f05b9fc7520, 0xc82087b800, 0xc820997a40, 0xc820900f60)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:251 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:254 +0xadc

goroutine 13902 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbbe48, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820052290, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820052290, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820052230, 0x0, 0x7f05b9fbbff0, 0xc820268860)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026b78, 0xc8205cad20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820873e30)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13897 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1591 +0xb48
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc820134b20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13906 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc8208ab740)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12881 [select]:
github.com/cockroachdb/cockroach/storage.(*Replica).requestLeaderLease(0xc82048bad0, 0x41a882851a, 0x8, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:438 +0x986
github.com/cockroachdb/cockroach/storage.(*Replica).redirectOnOrAcquireLeaderLease(0xc82048bad0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:482 +0x3e3
github.com/cockroachdb/cockroach/storage.(*Replica).addWriteCmd(0xc82048bad0, 0x7f05b9fbbfb0, 0xc8202d21b0, 0x41a882851a, 0x6, 0x600000006, 0x5, 0xb, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:903 +0x192
github.com/cockroachdb/cockroach/storage.(*Replica).Send(0xc82048bad0, 0x7f05b9fbbfb0, 0xc8202d21b0, 0x41a882851a, 0x6, 0x600000006, 0x5, 0xb, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:663 +0x614
github.com/cockroachdb/cockroach/storage.(*Store).Send(0xc8208a6c00, 0x7f05b9fbbfb0, 0xc8202d21b0, 0x41a882851a, 0x6, 0x600000006, 0x5, 0xb, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1288 +0xe14
--
testing.tRunner(0xc8201a8120, 0x2453840)
	/tmp/workdir/go/src/testing/testing.go:456 +0xdd
created by testing.RunTests
	/tmp/workdir/go/src/testing/testing.go:561 +0xaa4

goroutine 12921 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8203871e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13933 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269bc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13929 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:626 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269880)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13946 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc820835740, 0x7f05b9fc7520, 0xc820286780, 0xc8202ce600)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:341 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc820835740, 0x7f05b9fc7520, 0xc820286780, 0xc8202ce600)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:248 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0xa6c

goroutine 13948 [IO wait]:
net.runtime_pollWait(0x7f05b9fbbb48, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820578a70, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820578a70, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820578a10, 0xc820047000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820026588, 0xc820047000, 0x1000, 0x1000, 0x7f05b9fc77d0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc8202ce6c0)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 13932 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269ba0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13911 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc820834660, 0xece41fe6a, 0x31258db1, 0x279aa20, 0xc820510510, 0xc820139880, 0xc820546c30, 0x62773c)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc820546c30, 0xc820872300)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:214 +0xf6
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc820872450, 0xc82084ff40)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:221 +0x46b
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:216 +0x2fe
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc8208ab7e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12936 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8200dd4c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13935 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269c00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12882 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:213 +0x208
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8203878a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12893 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8204e71d0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 13941 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8208349c0, 0x7f05b9fc7520, 0xc820286000, 0xc8202ce360)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:341 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8208349c0, 0x7f05b9fc7520, 0xc820286000, 0xc8202ce360)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:248 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0xa6c

goroutine 13943 [IO wait]:
net.runtime_pollWait(0x7f05b9fbbd88, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820578610, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820578610, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc8205785b0, 0xc820384000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820026548, 0xc820384000, 0x1000, 0x1000, 0x7f05b9fc77d0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc8202ce420)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 13934 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820269be0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12889 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:566 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc820386a80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13915 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8208ab9c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12983 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbb848, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8209086f0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8209086f0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820908690, 0x0, 0x7f05b9fbbff0, 0xc820984680)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc8200261e0, 0xc820468d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8208b4ed0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12890 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:626 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc820386aa0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12967 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*idAllocator).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/id_alloc.go:129 +0xba3
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc8201b0500)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13916 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8208ab9e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13919 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc820834d20, 0xece41fe6a, 0x31a91e1e, 0x279aa20, 0xc820510510, 0xc820139d50, 0xc82048a340, 0x62773c)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc82048a340, 0xc8208730b0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:214 +0xf6
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc820873230, 0xc8205ac040)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:221 +0x46b
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:216 +0x2fe
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8208aba40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13950 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc820835080, 0x7f05b9fc7520, 0xc820286a80, 0xc8202ce7e0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:341 +0xb5
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc820835080, 0x7f05b9fc7520, 0xc820286a80, 0xc8202ce7e0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:248 +0x4d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0xa6c

goroutine 13954 [IO wait]:
net.runtime_pollWait(0x7f05b8740a10, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820578bc0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820578bc0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820578b60, 0xc8202f1000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc8200ea4d8, 0xc8202f1000, 0x1000, 0x1000, 0x7f05b9fc77d0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
net/rpc.(*Client).input(0xc8204c2420)
	/usr/local/go/src/net/rpc/client.go:109 +0x177
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0x126

goroutine 12896 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820386de0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13913 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8208ab960)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13922 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:626 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8202693a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13927 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1591 +0xb48
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc820119b50)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13857 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc8208ab700)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12984 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8200dca40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13930 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:645 +0x3ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8200522a0, 0xc8202698a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12920 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbb008, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820908220, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820908220, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc8209081c0, 0x0, 0x7f05b9fbbff0, 0xc82087e2e0)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc8200264a8, 0xc820251d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8204c5bf0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13910 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139880, 0xc8208ab7c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13925 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbaf48, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8200527d0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8200527d0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820052770, 0x0, 0x7f05b9fbbff0, 0xc820540da0)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026be8, 0xc82051dd20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc820584f30)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12907 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526a10, 0xc820513440)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 12886 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b9fbba88, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc8205278e0, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc8205278e0, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820527880, 0x0, 0x7f05b9fbbff0, 0xc820985b20)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026408, 0xc820038d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8204c5020)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13942 [IO wait]:
net.runtime_pollWait(0x7f05b9fbb608, 0x72, 0x0)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820578990, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820578990, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).Read(0xc820578930, 0xc820235000, 0x1000, 0x1000, 0x0, 0x7f05b9fb4050, 0xc8200121a0)
	/tmp/workdir/go/src/net/fd_unix.go:232 +0x27b
net.(*conn).Read(0xc820026558, 0xc820235000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/tmp/workdir/go/src/net/net.go:172 +0x121
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8208349c0, 0x7f05b9fc72b8, 0xc820026558, 0x7f05b9fc7520, 0xc820286000, 0xc820952d10, 0xc8202ce360)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:251 +0x6f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:254 +0xadc

goroutine 12895 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f05b8740ad0, 0x72, 0x7f05b9fb4050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc820527e20, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc820527e20, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc820527dc0, 0x0, 0x7f05b9fbbff0, 0xc820985520)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc820026458, 0xc820518d20, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820526690, 0xc8204c56b0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70

goroutine 13918 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820139d50, 0xc8208aba20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70
FAIL	github.com/cockroachdb/cockroach/storage	300.082s
=== RUN   TestBatchBasics
I0202 01:33:27.077150 701 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.00s)
=== RUN   TestBatchGet
I0202 01:33:27.081096 701 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.01s)
=== RUN   TestBatchMerge
I0202 01:33:27.088101 701 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.00s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

--- a/storage/client_split_test.go
+++ b/storage/client_split_test.go
@@ -764,7 +764,24 @@ func setupSplitSnapshotRace(t *testing.T) (mtc *multiTestContext, leftKey roachp
 	mtc.unreplicateRange(rightRangeID, 2)
 	mtc.unreplicateRange(rightRangeID, 1)
 
-	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 2, 2, 2})
+	// Perform another increment after all the replication changes. This
+	// lets us ensure that all the replication changes have been
+	// processed and applied on all replicas. This is necessary because
+	// the range is in an unstable state at the time of the last
+	// unreplicateRange call above. It has four members which means it
+	// can only tolerate one failure without losing quorum. That failure
+	// is store 3 which we stopped earlier. Stopping store 1 too soon
+	// (before it has committed the final config change *and* propagate
+	// that commit to the followers 4 and 5) would constitute a second
+	// failure and render the range unable to achieve quorum after
+	// restart (in the SnapshotWins branch).
+	incArgs = incrementArgs(rightKey, 3)
+	if _, pErr := client.SendWrapped(mtc.distSender, nil, &incArgs); pErr != nil {
+		t.Fatal(pErr)
+	}
+
+	// Store 3 still has the old value, but 4 and 5 are up to date.
+	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 2, 5, 5})
 
 	// Stop the remaining data stores.
 	mtc.stopStore(1)
@@ -807,7 +824,7 @@ func TestSplitSnapshotRace_SplitWins(t *testing.T) {
 	if _, pErr := client.SendWrapped(mtc.distSender, nil, &incArgs); pErr != nil {
 		t.Fatal(pErr)
 	}
-	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 22, 22, 22})
+	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 25, 25, 25})
 }
 
 // TestSplitSnapshotRace_SplitWins exercises one outcome of the
@@ -839,7 +856,7 @@ func TestSplitSnapshotRace_SnapshotWins(t *testing.T) {
 	// for. There is a high probability that the message will have been
 	// received by the time that nodes 4 and 5 have processed their
 	// update.
-	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 2, 22, 22})
+	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 2, 25, 25})
 
 	// Wake up the left-hand range. This will allow the left-hand
 	// range's split to complete and unblock the right-hand range.
@@ -859,7 +876,7 @@ func TestSplitSnapshotRace_SnapshotWins(t *testing.T) {
 	if _, pErr := client.SendWrapped(mtc.distSender, nil, &incArgs); pErr != nil {
 		t.Fatal(pErr)
 	}
-	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 222, 222, 222})
+	mtc.waitForValues(rightKey, splitTimeout, []int64{0, 0, 0, 225, 225, 225})
 }
 
 // TestStoreSplitReadRace prevents regression of #3148. It begins a couple of

